operator/node.go

```diff
@@ -52,8 +52,8 @@ type operatorNode struct {
 	storage             Storage
 	genesisEpoch        uint64
 	dutyLimit           uint64
-	streamDuties        <-chan *ethpb.DutiesResponse_Duty
 	eth1Client          eth1.Client
+	epochDuties         bool
```

Comment:
@nivBlox what is the purpose of this?

Reply:
we need some indication of when we already fetched duties. 
could be change to better solution + will add explanation

Reply:
@nivBlox please change to EpochDutiesExist

Reply:
done

Reply:
@nivBlox if you'd want to refactor it in the future please add a TODO comment and an issue so we know to get back to it

Reply:
https://github.com/bloxapp/ssv/issues/179